README.md

```diff
@@ -12,15 +12,9 @@ But that is just to help you get started! To make it your own:
 - [ ] Optional: Add a `logo.png` file to give this plugin its own logo.
 - [ ] Create a README.MD for your plugin which follows [our documentation guidelines](https://posthog.com/docs/plugins/build). 
 
-If you're looking for inspiration, here are a few exemplary plugins:
+If you're looking for inspiration, then check out plugins source code from [data pipelines](https://posthog.com/docs/cdp).
```

Comment:
Hmm, this link doesn't seem to go anywhere all that useful? I don't have a better idea off the top of my head though. Maaaaybe https://github.com/PostHog/?q=plugin&type=all&language=&sort= ?

Reply:
agreed, I was thinking they can check out plugins there, which have links to repos - which allows them to read what the plugin does first, pick if they are looking to do a destination or ingestion filtering or event transformation on the left, so in theory better than just linking to a mix of plugins
```suggestion
If you're looking for inspiration, then check out existing plugins and their source code from [data pipelines](https://posthog.com/docs/cdp).
```